### ISSUE TITLE
SVL MAPS are not available on Dreamview/Apollo6.0 ?

### ISSUE THREAD

**Issue by marcusvinicius178** [ApolloAuto/apollo#13774]
Hi I Dowloaded **SanFrancisco map** in SVL simulator.
**I would like to use it with Apollo 6.0**. However when I open Dreamview:

![maps_available](https://user-images.githubusercontent.com/43826438/118157064-77e5bd80-b3f0-11eb-9ced-85452b7e9484.png)

I cannot locate it. There **are just the BorregasAV map avalable and the Sunnyloop** derivatives map available. I cannot see in dreamview any maps from those available on SVL simulator. 

**How to use SanFranscisco map or anothers maps from those given into SVL within Apollo 6.0?**

![maps_SVL](https://user-images.githubusercontent.com/43826438/118157387-dd39ae80-b3f0-11eb-8a11-a7d87223d0d7.png)

Thanks

**Comment by Eclipsehelio** [comment]
Have you put the map in `/apollo/modules/map/data` directory?

**Comment by marcusvinicius178** [comment]
Hi. No I still didn't. But these maps from SVL? I mean where can I get these SVL maps folder with all the content inside? 
Afterwards is just copy and paste into this path you described? Or is it required to build/compile the workspace again?

**Comment by marcusvinicius178** [comment]
Hi I found the other maps from SVL in older Apollo Release (Apollo 5.0)
Then I have copied and pasted into the map/data folder in Apollo 6.0.

![image](https://user-images.githubusercontent.com/43826438/118675738-3890e580-b7d1-11eb-9ab8-3cb0f8082901.png)

Afterwards compiled again the workspace:
`./apollo.sh build_opt_gpu`

Then finally I ebabled the maps on dreamview and the simulator opened the scenario.

![Captura de tela de 2021-05-18 12-06-27](https://user-images.githubusercontent.com/43826438/118676206-98878c00-b7d1-11eb-9a5d-f029dc4d0671.png)

**HOWEVER AFTER 5 SECONDS THE SIMULATOR IS SHUTDOWN. THEREFORE I CONCLUDE THAT JUST BORREGASAVE MAP IS CURRENTLY WORKING ON APOLLO6.0, CORRECT?**
